files/scripts/syncd.sh

```diff
@@ -36,12 +36,7 @@ function startplatform() {
         debug "Starting Firmware update procedure"
         /usr/bin/mst start --with_i2cdev
 
-        local -r _MST_DEVICE="$(GetMstDevice)"
```

Comment:
We can remove GetMstDevice method from the syncd.sh i suppose

Reply:
Done.